poetry.lock

```diff
@@ -8,36 +8,36 @@ python-versions = "*"
 
 [[package]]
 name = "attrs"
-version = "21.4.0"
+version = "22.1.0"
```

Comment:
I didn't realize we were keeping a `poetry.lock` file in-repo. For a library this surprises me....